clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -2924,7 +2966,40 @@ void ConvertCIRToLLVMPass::runOnOperation() {
   getOperation()->removeAttr("cir.sob");
   getOperation()->removeAttr("cir.lang");
 
-  if (failed(applyPartialConversion(module, target, std::move(patterns))))
+  llvm::SmallVector<mlir::Operation*> ops;
```

Comment:
here is the stub I was talking about - and I don't like it. I collect unreachable blocks and then iterate them, discovering successors, add theirs operations and etc.